components/HttpClient.tsx

```diff
@@ -0,0 +1,72 @@
+import { Alert } from "react-native";
+import ApiDictionary from '../constants/ApiDictionary';
+
+const state={
+    jwt:"",
+}
+
+export default async function bodyless(details: { destination: string; type: string;}) {
+
+    const response = await fetch(ApiDictionary.apiServer + details.destination , {
+        method: details.type,
+        headers: {
+            'Content-Type': 'application/json',
+            'jwt': state.jwt,
```

Comment:
Moest het niet 'authorization' zijn ipv 'jwt' voor de backend

Reply:
Oh en nog die code die in de .env staat achter JWT_KEY

Reply:
Hier zou ik niet zo 123 vanaf weten. Klinkt als een taakje voor de auth userstory🤭